### ISSUE TITLE
Hyphenate long words on mobile

### ISSUE THREAD

**Issue by mlbrgl** [owid/owid-grapher#1363]
## Core problem

_I'm checking some of the pages on mobile for formatting issues. On the covid cases page for sequences by variant, the title in the header saying Acknowledgements on the same page is too long as a single word and runs onto the next line without an (automated?) hyphen. This might affect other pages with long words in the headers._ - @saloni-nd 

## Proposed solution

Automated hypens for words than wrap onto next line via CSS

## Context

https://ourworldindata.org/covid-cases#acknowledgements
![Screenshot_20220304_121300](https://user-images.githubusercontent.com/13406362/161715151-b240a69c-5716-45ed-9d57-3813fd10d995.jpg)

**Comment by marcelgerber** [comment]
Context: https://developer.mozilla.org/en-US/docs/Web/CSS/hyphens

**Comment by YannyTwoo** [comment]
Can I be assigned this issue? Wish to start contributing but not sure if it conflicts with the website design restrictions on contributing.md

**Comment by YannyTwoo** [comment]
Implementation for proposed solution-
Since the website is wordpress based it seems best to use the [wp-Typography plugin](https://wordpress.org/plugins/wp-typography/), this will add the necessary hyphenation points to the content that is to be rendered.
Since the actual hyphenation is done by the browser, we can add the CSS property of "hyphens:manual" to all the h-tags which are likely to have characters exceeding the mobile size in the css file in /web. ( Can also add "hyphens:none" and have it not apply to any non h-tag to avoid excessive hyphenation)

**Comment by mlbrgl** [comment]
hi @YannyTwoo, sorry for the delay!

Thanks for your suggestion, that sounds sensible. We'll be moving away from Wordpress soon so we probably won't rely on a WP-centric solution for this, but the CSS part sounds good.

Do you mind carving out a PR with the `auto` alternative? We probably also need to add a bunch of missing `lang` attributes to our `<html>` tags.

**Comment by mlbrgl** [comment]
@YannyTwoo on second thoughts, let me confirm what we want to hyphenate first. This might be wider ranging (across paragraphs, etc... and responsive sizes).

**Comment by mlbrgl** [comment]
@YannyTwoo discussing this some more internally made us realize that aligning older pages with the new (smaller) typography styles would likely solve this problem for the most part. #1859 is where this work is tracked.

I'm sorry we ended up turning around here after you spent some time thinking about this and coming up with a viable solution. I appreciated your input and contribution nevertheless.

I'll close this issue for now, but I'll keep an eye out for similar issues in the future and might tag you if you're interested.

👋
